criu/proc_parse.c

```diff
@@ -1407,7 +1407,7 @@ static int parse_mountinfo_ent(char *str, struct mount_info *new, char **fsname)
 		goto err;
 
 	new->mountpoint[0] = '.';
-	ret = sscanf(str, "%i %i %u:%u %ms %s %ms %n", &new->mnt_id, &new->parent_mnt_id, &kmaj, &kmin, &new->root,
+	ret = sscanf(str, "%i %i %u:%u %ms %4094s %ms %n", &new->mnt_id, &new->parent_mnt_id, &kmaj, &kmin, &new->root,
```

Comment:
All these constants  don't look good. Can we generate them with __stringify?

Reply:
AFAICS there is no reasonable way to do a macro to stringify PATH_MAX-2 result, but we can probably do:
```
#define PATH_MAX_MINUS_2 4094
```
to remove magic numbers.

Reply:
Do I need to push some new patches?